modules/light-clients/06-solomachine/misbehaviour_handle.go

```diff
@@ -19,7 +19,7 @@ func (cs ClientState) CheckForMisbehaviour(_ sdk.Context, _ codec.BinaryCodec, _
 	return false
 }
 
-func (cs ClientState) verifyMisbehaviour(ctx sdk.Context, cdc codec.BinaryCodec, clientStore sdk.KVStore, misbehaviour *Misbehaviour) error {
+func (cs ClientState) verifyMisbehaviour(cdc codec.BinaryCodec, clientStore sdk.KVStore, misbehaviour *Misbehaviour) error {
```

Comment:
guess `unused-parameter` being removed complained about this? Looks like a right idea but again I'd think its better if reviewed separately.

Reply:
totally agree, I'll redo as described :)